osf/metrics.py

```diff
@@ -0,0 +1,129 @@
+from elasticsearch_metrics import metrics
+from django.db import models
+
+class MetricMixin(object):
```

Comment:
It's kinda confusing to have two `MetricMixin`s, at `osf.metrics.MetricMixin` and `api.base.metrics.MetricMixin`